<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coffee</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1071</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addOrEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>355</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить кофе</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
